src/app/app.component.html

```diff
@@ -9,21 +9,9 @@
   <tour-step-template>
     <ng-template let-step="step">
       <p class="tour-step-content">{{step?.content}}</p>
-      <div class="tour-step-navigation">
-        <button *ngIf="tourService.hasPrev(step)" class="btn btn-sm btn-default" (click)="tourService.prev()">« Prev</button>
-        <button *ngIf="tourService.hasNext(step)" class="btn btn-sm btn-default" (click)="tourService.next()">Next »</button>
-        <!--
-        <button class="btn btn-sm btn-default" (click)="tourService.end()">End</button>
-        <button (click)="tourService.end()"
-                class="close end-tour pull-right">
-          <span class="pficon pficon-close"></span>
-        </button>
-        <!--
-        <button (click)="tourService.end()"
-                class="close end-tour">
-          <span class="pficon pficon-close"></span>
-        </button>
-        -->
+      <div class="tour-step-navigation text-right">
+        <button *ngIf="tourService.hasPrev(step)" class="btn btn-sm btn-primary" (click)="tourService.prev()">« Prev</button>
```

Comment:
Should these attributes be split out into separate lines?

Reply:
Yes

Reply:
You know, lemme get an issue open for HTML formatting, maybe we can make a pre-commit hook for something there too...